--- a/Makefile	2016-01-23 17:34:57.000000000 -0600
+++ b/Makefile1	2016-04-19 23:53:01.684266546 -0600
@@ -749,7 +749,7 @@
 codec-cfg-test$(EXESUF): HOSTCFLAGS := $(HOSTCFLAGS) -DTESTING
 codec-cfg$(EXESUF) codecs2html$(EXESUF):  HOSTCFLAGS := $(HOSTCFLAGS) -DCODECS2HTML
 codec-cfg$(EXESUF) codec-cfg-test$(EXESUF) codecs2html$(EXESUF): codec-cfg.c codec-cfg.h help_mp.h
-	$(HOST_CC) $(HOSTCFLAGS) -o $@ $<
+	$(HOST_CC) $(HOSTCFLAGS) $(CFLAGS)-o $@ $<
 
 codecs.conf.h: codec-cfg$(EXESUF) etc/codecs.conf
 	./$^ > $@
@@ -868,10 +868,10 @@
 install-mplayer-man:  $(foreach lang,$(MAN_LANGS),install-mplayer-man-$(lang))
 
 install-gui-man-en: install-mplayer-man-en
-	cd $(MANDIR)/man1/ && ln -sf mplayer.1 gmplayer.1
+	echo ".so mplayer.1" > $(MANDIR)/man1/gmplayer.1
 
 install-mencoder-man-en: install-mplayer-man-en
-	cd $(MANDIR)/man1 && ln -sf mplayer.1 mencoder.1
+	echo ".so mplayer.1" > $(MANDIR)/man1/mencoder.1
 
 install-mplayer-man-en:
 	$(INSTALL) -d $(MANDIR)/man1
@@ -885,12 +885,12 @@
 
 define GUI_MAN_RULE
 install-gui-man-$(lang): install-mplayer-man-$(lang)
-	cd $(MANDIR)/$(lang)/man1/ && ln -sf mplayer.1 gmplayer.1
+	echo ".so mplayer.1" > $(MANDIR)/$(lang)/man1/gmplayer.1
 endef
 
 define MENCODER_MAN_RULE
 install-mencoder-man-$(lang): install-mplayer-man-$(lang)
-	cd $(MANDIR)/$(lang)/man1 && ln -sf mplayer.1 mencoder.1
+	echo ".so mplayer.1" > $(MANDIR)/$(lang)/man1/mencoder.1
 endef
 
 define MPLAYER_MAN_RULE
